src/index.test.ts

```diff
@@ -2,9 +2,8 @@ import fs from 'node:fs'
 import path from 'node:path'
 
 import { describe, it, expect, vi, beforeEach, afterEach, beforeAll, afterAll } from 'vitest'
-import { addAttachment, addArgument } from '@wdio/allure-reporter'
 import { browser } from '@wdio/globals'
-
+import AllureReporterExtension from './allure.js'
```

Comment:
I changed it to spy on the allureExtention rather than actual allure functions because since they were async, it was flakey